### ISSUE TITLE
Display list of Ads/Trackers | HTTPS Upgrades | Blocked Scripts and FP 

### ISSUE THREAD

**Issue by jumde** [brave/brave-ios#764]
### Description: 
Display list of Ads/Trackers, HTTPS Upgrades, blocked scripts and fingerprinting urls in the shields panel

**Comment by Eyesonly88** [comment]
Yes, the mobile version is missing the ability to know which sites have been https upgraded and so on. It would be really useful to know this information.
